impl/src/main/java/com/okta/sdk/impl/resource/DefaultUserBuilder.java

```diff
@@ -268,6 +268,10 @@ public UserBuilder setSha1PasswordHash(String value, String salt, String saltOrd
         return setShaPasswordHash("SHA-1", value, salt, saltOrder);
     }
 
+    public UserBuilder setMd5PasswordHash(String value, String salt, String saltOrder) {
+        return setShaPasswordHash("MD5", value, salt, saltOrder);
```

Comment:
Nit: `MD5` is technically different from `SHA` algorithm. Calling `setShaPasswordHash("MD5",...)` here looks odd.